.travis.yml

```diff
@@ -14,10 +14,10 @@ script:
   - phpunit --coverage-clover=coverage.clover
 
 before_script:
-  - npm install -g bower
-  - composer install
   - mysql -e 'create database fork_test'
   - mysql -u travis fork_test < tools/test_db.sql
+  - npm install -g bower
+  - composer install
```

Comment:
does this mean that we can get issues locally when setting up fork before creating the database?